isympy.py

```diff
@@ -173,7 +173,7 @@
 # DO NOT IMPORT SYMPY HERE! Or the setting of the sympy environment variables
 # by the command line will break.
 
-def main():
+def main() -> None:
     from argparse import ArgumentParser, RawDescriptionHelpFormatter
```

Comment:
I don't see the point of making this change.

Reply:
If you feel like this is pointless we can close this.

Reply:
Okay, I'll close this.